ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_service_insertion.schema.yml

```
@@ -8,7 +8,41 @@ type: dict
 keys:
   router_service_insertion:
     type: dict
-    description: Configure network services inserted to data forwarding
+    description: Configure network services inserted to data forwarding.
     keys:
       enabled:
         type: bool
+      connections:
+        type: list
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Configure name of the network service.
+            interface:
+              type: dict
+              description: Interface to be used to reach the network service.
+              keys:
+                ethernet:
+                  type: dict
+                  description: Configure to use ethernet interface to reach the network service.
+                  keys:
+                    interface_name:
+                      type: str
+                      description: e.g. ethernet2
+                    next_hop_ip_address:
+                      type: str
+                      description: Configure the next-hop ip address.
+                tunnel:
+                  type: dict
+                  description: Configure to use tunnel interface to reach the network service.
+                  keys:
+                    primary_interface_name:
+                      type: str
+                      description: e.g. tunnel2
+                    secondary_interface_name:
+                      type: str
+            monitor_connectivity_to_host:
+              type: str
+              description: Monitor connectivity health to the network service host.
```

Please double check the path
```suggestion
              description: |-
                Monitor connectivity host to use to derive the health of the connecion.
                The host must be defined under `monitor_connectivity.hosts`.
```

---

Corrected.